test/views/CannotClaimEmployerContributionViewSpec.scala

```diff
@@ -17,7 +17,6 @@
 package views
```

Comment:
Can we add an application stop on this

Reply:
None of the rest of the view specs have application.stop()